### PR TITLE
Rework puclic API - better ui config in bucket.init()

### PR DESCRIPTION
**Pull request by Munter**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -39,12 +58,60 @@ export type TrackedEvent = {
...
+  featureId: string;
+  userId: string;
+  companyId?: string;
+
+  /**
+   * Allows you to handle a copy of the already submitted
+   * feedback.
+   *
+   * This can be used for side effects, such as storing a
+   * copy of the feedback in your own applicaiton or CRM.
```
Comment (pavkam) [inline src/types.ts:72]: `applicaiton` -> `application`

File: src/types.ts [hunk 2]
```diff
@@ -39,12 +58,60 @@ export type TrackedEvent = {
...
+}
+
 export type Feedback = {
+  /**
+   * Bucket feature ID
+   */
   featureId: string;
+
+  /**
+   * User id from your own appliction
```
Comment (pavkam) [inline src/types.ts:87]: `appliction` -> `application`

**Review by pavkam — APPROVED** [review]

File: src/types.ts (hunk 1 above)
Comment (Munter) [reply src/types.ts:72]: I'll fix those before cutting a release

File: src/types.ts (hunk 2 above)
Comment (Munter) [reply src/types.ts:87]: I'll fix those before cutting a release

**Pull request merged**